profiles/permissions.py

```diff
@@ -15,3 +16,15 @@ class ReadOnly(BasePermission):
     def has_permission(self, request, view):
         if request.method in SAFE_METHODS:
                  return True
+
+
+class SelfCompany(BasePermission):
```

Comment:
Please consider better name for this permission. Probably something starting with `Is...`, `Has...` etc.

Reply:
Thank you for feedback. Done.